transposon/genotypeClones.py

```diff
@@ -144,6 +144,7 @@ def pruneOrphanNodes(G):
 def remove_edges(G, edgesToRemove):
     #Convert to set to speed-up inclusion check
     edgesToRemove = set(edgesToRemove)
+    edgesToRemove.update([(y, x) for (x, y) in edgesToRemove])
```

Comment:
Andre, can you add a comment to expand on what this is doing? This might be better done in a separate PR so it doesn't show up with the unrelated commit below.

Reply:
I've committed them separately, but do you want me to convert this to two distinct PR?

Reply:
yes, in the future it's best to make separate PRs. That way I can squash and merge regardless of how many commits happen.